src/main/include/RobotContainer.h

```diff
@@ -15,12 +16,10 @@ class RobotContainer {
   RobotContainer();
 
   static RobotContainer* Get();
-
   std::shared_ptr<t34::SwerveDrive> swerve_drive;
   std::shared_ptr<t34::T34CommandXboxController> ctrl;
-
+  t34::Intake intake;
```

Comment:
Unless there is a good reason to keep this public, we should probably make it private. Also, naming convention:

```suggestion
  t34::Intake m_intake;
```

Reply:
👍